### ISSUE TITLE
DBN regression

### ISSUE THREAD

**Issue by JIAZhenZ** [Jxinyee/dbn_pytorch#2]
I wonder to know that how should I revise the code if I want to solve a regression question by DBN? Looking forward for your answer, tanks.

**Comment by Jxinyee** [comment]
you can rewrite the softmax activate function. use a full-connected  node to replace it  and  'mse' is the loss function
